notebooks/quasar_population_tutorial.ipynb

```diff
@@ -2,15 +2,18 @@
  "cells": [
```

Comment:
<p>is this needed? If so, perhaps having a proper print statement?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think it is not needed.</p><p> </p>